internal/npm_install/npm_install.bzl

```diff
@@ -381,23 +374,11 @@ def _yarn_install_impl(repository_ctx):
 
 yarn_install = repository_rule(
     attrs = dict(COMMON_ATTRIBUTES, **{
-        "timeout": attr.int(
-            default = 3600,
-            doc = """Maximum duration of the command "yarn install" in seconds
-            (default is 3600 seconds).""",
-        ),
-        "frozen_lockfile": attr.bool(
-            default = False,
-            doc = """Passes the --frozen-lockfile flag to prevent updating yarn.lock.
+        "args": attr.string_list(
+            doc = """Arguments to passed to the yarn.
```

Comment:
"the yarn" what?

Reply:
Thanks. Fixed to "Arguments passed to yarn install."